### PR TITLE
Minor UI fixes and improvements

### PR DESCRIPTION
**Pull request by sloria**
Various small UI improvements.

### CODE REVIEW

File: client/app/bundles/Room/components/Story.jsx [hunk 1]
```diff
@@ -35,11 +35,23 @@ export default class Story extends React.Component {
       }
     })()
 
+    const storyTitle = (() => {
```
Comment (sloria) [inline client/app/bundles/Room/components/Story.jsx]: This is a bugfix. Previously, story titles would get turned into links even if they weren't URLs. With this fix, only links that start with `"http"` will be linkified.

File: client/app/libs/barColors.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ const barColors = {
 
 const pointEmojis = {
   "coffee": "☕",
-  "?": "⁉️",
+  "?": "?",
```
Comment (hlcfan) [inline client/app/libs/barColors.js]: Since "question mark" isn't a emoji char, please remove this line.

File: app/views/rooms/_form.html.erb [hunk 3]
```diff
@@ -71,10 +71,6 @@
       </button>
     </div>
   </section>
-  <div class="form-group">
-    <%= f.label "Moderators" %>
-    <%= f.text_field :moderator_ids, class: "form-control room--moderators"  %>
-  </div>
```
Comment (hlcfan) [inline app/views/rooms/_form.html.erb]: Can you keep this? One of my friends wants to add moderators while creating room 😄

Comment (sloria) [reply app/views/rooms/_form.html.erb]: Sure. Can you explain what the expected input is here? Is it a comma-separated list of names?

Once I understand what is expected here, I can add a placeholder with an explanation and/or example input.

Comment (hlcfan) [reply app/views/rooms/_form.html.erb]: It's an autocomplete input, see https://github.com/hlcfan/pokr/pull/101

Comment (sloria) [reply app/views/rooms/_form.html.erb]: Hm, the placeholder is getting clipped for some reason, so I'm going to hold off on adding it for now

create room pokrex 2017-10-07 21-37-14

**Comment by sloria** [comment]
@hlcfan OK, I've made the changes you requested.

**Comment by hlcfan** [comment]
79a00895ly1fb7fqay7bgg208c04pqv50 1

**Pull request merged**